addons/mail_group/tests/test_mail_group_message.py

```diff
@@ -44,6 +44,23 @@ def test_batch_send(self):
         # 42 -1 as the sender doesn't get an email
         self.assertEqual(len(mails), 41, 'Should have send one and only one email per recipient')
 
+    def test_email_bounce(self):
```

Comment:
Would probably go for a "test_group_closed", as it is the real feature we check :)